libs/core/led.ts

```diff
@@ -21,24 +21,28 @@ namespace led {
     let barGraphHigh = 0;
     // when was the current high value recorded
     let barGraphHighLast = 0;
-    
+
     /**
      * Controls where plotbargraph prints to the console
-     **/ 
+     **/
     export let barGraphToConsole = true
 
     /**
      * Displays a vertical bar graph based on the `value` and `high` value.
      * If `high` is 0, the chart gets adjusted automatically.
      * @param value current value to plot
      * @param high maximum value. If 0, maximum value adjusted automatically, eg: 0
+     * @param valueToConsole if true, prints value to the console
      */
     //% help=led/plot-bar-graph weight=20
-    //% blockId=device_plot_bar_graph block="plot bar graph of %value up to %high" icon="\uf080" blockExternalInputs=true
+    //% blockId=device_plot_bar_graph block="plot bar graph of %value up to %high|| console log %valueToConsole" icon="\uf080" blockExternalInputs=true
```

Comment:
```suggestion
    //% blockId=device_plot_bar_graph block="plot bar graph of $value up to $high|| console log $valueToConsole" icon="\uf080" blockExternalInputs=true
```

if we're changing the block definition anyways we should change them to $ -- the only reason we left them as is is to avoid messing up the translations, and if we're changing the string the translations won't carry over anyways